src/main/java/auth/TdpRecruitmentAuthorizer.java

```diff
@@ -0,0 +1,19 @@
+package auth;
+
+
+import domain.Person;
+import io.dropwizard.auth.Authorizer;
+
+public class TdpRecruitmentAuthorizer implements Authorizer<Person> {
+	@Override
+	public boolean authorize(Person principal, String role) {
+
+		if(role.equals("recruiter")) {
+			return principal.getAdmin() == true ? true : false;
+		} else if(role.equals("interviewer")) {
+			return principal.getAdmin() == false ? true : false;
```

Comment:
You do `return principal.getAdmin() == true` and `return principal.getAdmin() == false`.